### DISCUSSION TITLE
What have / are you learning in December?

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#510] [Poll]
Put your topics below and vote on other peoples 🤓

You can post multiple times as the month goes on 👇

**Comment by adityaraute** [comment]
Flask 🤓

**Reply by eddiejaoude** [reply]
Awesome, looking forward to the update in Jan 🤓

**Comment by BOLT04** [comment]
Azure Fundamentals 😃

**Reply by eddiejaoude** [reply]
Oh nice, let me know how it goes! I tried it briefly, it seemed good, but pricing seemed a bit crazy. For the small EddieBot, it burnt through £200 in 2 months 😱

**Reply by BOLT04** [reply]
oh wow, did you use Azure App Service or a serverless service, like Functions?
As far as pricing goes and free tier, I've had a really good experience in GCP and Heroku 😃

**Comment by Raheel92** [comment]
Python (Flask + Scripting), AWS and Linux

**Reply by eddiejaoude** [reply]
Awesome, that will keep you busy 👍

**Comment by mikeysan** [comment]
I think @Raheel92 and I have a similar path, minus the Flask bit. 
Python, AWS, Postman (REST API), and Linux (oh yeah, and job hunting) 😃

**Reply by adityaraute** [reply]
Oh job hunting is a biggie, ain't it? 😢

**Reply by mikeysan** [reply]
It really is. But we do what we gotta do, right.

**Reply by eddiejaoude** [reply]
Awesome list 💪 ! @mikeysan has got this 🤓

**Reply by Raheel92** [reply]
That's nice to know @mikeysan, after learning the concepts what kinds of projects are you going to be working on? Probably we can collaborate. I am also going on job hunting after a month or so.

**Comment by hasthamalp** [comment]
React native

**Reply by eddiejaoude** [reply]
Let us know what you build with it 🤓

**Reply by emmysteven** [reply]
Wow, that's nice

**Comment by jasontwuk** [comment]
Github, git, Markdown and JavaScript Algorithms.

**Reply by eddiejaoude** [reply]
Nice 👍 - I have `markdown` video coming out soon 🤓 . Let me know what topics you are or want to cover in each and I can try to do videos on it

**Reply by jasontwuk** [reply]
I didn’t realise `.md` files in Github were `markdown` until I watched your [Complete Guide to Open Source - How to Contribute](https://www.youtube.com/watch?v=yzeVMecydCE) video on Youtube. In the dark time, I used `HTML` to code the `.md` files.🙈 For a beginner, I think it will be nice to know where you can use `markdown` and why we should use it. Thank you. 🙂

**Reply by eddiejaoude** [reply]
You are welcome 🤓

**Reply by jasontwuk** [reply]
I made a markdown cheat sheet from what I have learnt lately. 
Any feedback is welcome. 🙂
https://github.com/jasontwuk/markdown-cheat-sheet

**Comment by mstanciu552** [comment]
Open source in general(Github) and refreshing my Typescript knowledge.
Also I've recently started solving problems from sites like [HackerRank](https://www.hackerrank.com/) and [leetcode](https://leetcode.com/).
Hope I have a productive December. 😄

**Reply by adityaraute** [reply]
Tried Advent of Code?

**Reply by eddiejaoude** [reply]
Yes ... open source 👍 typescript 👍

**Reply by mstanciu552** [reply]
I want to try advent of code
It's just that I have to find the time.
:yum:

**Comment by eddiejaoude** [comment]
Oh I didn't share mine yet 🤦‍♂️ 

Learning
- NuxtJS over the festive season
- Digital Ocean with Docker

**Comment by fidotheprince** [comment]
Learning PHP and MySQL 🙈

**Reply by eddiejaoude** [reply]
Yay 🎉

**Comment by Akshu-on-github** [comment]
Django and Discord.py
(( Very python heavy, I know ;P ))

**Reply by emmysteven** [reply]
🎉

**Reply by eddiejaoude** [reply]
Sounds good, doubling down 👍

**Comment by iamkennis** [comment]
Hi, am Leaning React Redux and Thunk

**Reply by emmysteven** [reply]
We have at least one thing in common 🎉

**Reply by eddiejaoude** [reply]
yay 🎉

**Comment by emmysteven** [comment]
Hi, I'm picking up React alongside with .NET Core

**Reply by eddiejaoude** [reply]
Cool 👍

**Comment by Vyvy-vi** [comment]
C, CSS, working on a few projects.
(and learning school stuff :/ )

**Reply by Akshu-on-github** [reply]
Keep going - Good buildings have strong foundations!
